contracts/go/voter/src/handler.go

```diff
@@ -300,9 +301,10 @@ func handleReplyBankMsg(deps *std.Deps, reply stdTypes.SubcallResult) (*stdTypes
 	}
 
 	var releasedAmt []stdTypes.Coin
+	fmt.Println(string(reply.Ok.Data))
```

Comment:
fmt.Println shouldn't work on the VM I think, API.Debug should be used.

Reply:
Agree, however ill be closing this PR in favor of #231